test/videojs.ads.test.js

```diff
@@ -1,1025 +1,732 @@
-  equal(player.ads.state, 'ads-ready');
-  player.trigger('play');
-  player.trigger('adtimeout');
-  cbs.pop().call(window);
-  player.trigger('ended');
-  equal(player.ads.state, 'postroll?');
-
-  player.ads.snapshot.ended = true;
-  player.trigger('adtimeout');
-
-  equal(player.ads.state, 'content-resuming');
-  equal(player.ads.triggerevent, 'adtimeout', 'adtimeout should be the trigger event');
-
-  cbs.pop().call(window);
-
-  equal(contentPlaybackFired, 2, 'A content-playback event should have been triggered');
-  equal(contentPlaybackReason, 'ended', 'The reason for content-playback should have been ended');
-  equal(player.ads.state, 'content-playback');
-
-  window.setImmediate = oldimmediate;
-});
-
-test('adskip in postroll? transitions to content-playback and fires ended', function(){
-  var oldimmediate = window.setImmediate,
-      cbs = [];
-  window.setImmediate = function(cb) {
-    cbs.push(cb);
-  };
-
-  equal(player.ads.state, 'content-set');
-  player.trigger('adsready');
-  equal(player.ads.state, 'ads-ready');
-  player.trigger('play');
-  player.trigger('adtimeout');
-  cbs.pop().call(window);
-  player.trigger('ended');
-  equal(player.ads.state, 'postroll?');
-
-  player.ads.snapshot.ended = true;
-  player.trigger('adskip');
-
-  equal(player.ads.state, 'content-resuming');
-  equal(player.ads.triggerevent, 'adskip', 'adskip should be the trigger event');
-
-  cbs.pop().call(window);
-
-  equal(contentPlaybackFired, 2, 'A content-playback event should have been triggered');
-  equal(contentPlaybackReason, 'ended', 'The reason for content-playback should have been ended');
-  equal(player.ads.state, 'content-playback');
-
-  window.setImmediate = oldimmediate;
-});
-
-test('an ended event is fired in content-resuming via a timeout if not fired naturally', function() {
-  var oldtimeout = window.setTimeout,
-      oldclear = window.clearTimeout,
-      ended = 0,
-      cbs = [];
-
-  window.setTimeout = function(cb) {
-    return cbs.push(cb) - 1;
-  };
-  window.clearTimeout = function(index) {
-    cbs.splice(index, 1);
-  };
-
-  player.on('ended', function() {
-    ended++;
-  });
-
-  equal(player.ads.state, 'content-set');
-  player.trigger('adsready');
-  equal(player.ads.state, 'ads-ready');
-  player.trigger('play');
-  player.trigger('adtimeout');
-  player.trigger('ended');
-  equal(player.ads.state, 'postroll?');
-
-  player.ads.startLinearAdMode();
-  player.ads.snapshot.ended = true;
-  player.ads.endLinearAdMode();
-
-  equal(player.ads.state, 'content-resuming');
-
-  equal(ended, 0, 'we should not have gotten an ended event yet');
-
-  if (!/phantomjs/i.test(window.navigator.userAgent)) {
-    cbs.pop().call(window);
-    equal(ended, 1, 'we should have fired ended from the timeout cbs');
-  }
-
-  window.setTimeout = oldtimeout;
-  window.clearTimeout = oldclear;
-});
-
-test('an ended event is not fired in content-resuming via a timeout if fired naturally', function() {
```

Comment:
Yeah, looks like I thought it was duplicated since it's so similar to the previous one. Bringing it back!